hsv_filter:
  ros__parameters:
    use_sim_time: false
    fine_tune: false
    min_h: 93
    max_h: 160
    min_s: 223
    max_s: 255
    min_v: 186
    max_v: 255
    use_ipc: true
    real_time: false

follow_controller:
  ros__parameters:
    pan_joint_name: "head_1_joint"
    tilt_joint_name: "head_2_joint"
    pan_joint_frame: "head_1_link"
    tilt_joint_frame: "head_2_link"
    camera_optical_frame: "head_front_camera_rgb_optical_frame"
    use_sim_time: false
    pan_pid_min_ref: 0.0
    pan_pid_max_ref: 1.0
    pan_pid_min_output: 0.0
    pan_pid_max_output: 0.3
    tilt_pid_min_ref: 0.0
    tilt_pid_max_ref: 1.0
    tilt_pid_min_output: 0.0
    tilt_pid_max_output: 0.3
    pan_limit: 1.3
    tilt_limit: 0.92
    action_ms: 1000
    use_ipc: true
    real_time: false

depth_estimator:
  ros__parameters:
    use_sim_time: false
    use_ipc: true
    real_time: false